libraries/AP_Logger/LogStructure.h

```diff
@@ -1320,7 +1321,8 @@ LOG_STRUCTURE_FROM_VISUALODOM \
     { LOG_PSC_MSG, sizeof(log_PSC), \
       "PSC", "Qffffffffffff", "TimeUS,TPX,TPY,PX,PY,TVX,TVY,VX,VY,TAX,TAY,AX,AY", "smmmmnnnnoooo", "F000000000000", true }, \
     { LOG_PSCZ_MSG, sizeof(log_PSCZ), \
-      "PSCZ", "Qfffffffff", "TimeUS,TPZ,PZ,DVZ,TVZ,VZ,DAZ,TAZ,AZ,ThO", "smmnnnooo%", "F000000002", true }
+      "PSCZ", "Qfffffffff", "TimeUS,TPZ,PZ,DVZ,TVZ,VZ,DAZ,TAZ,AZ,ThO", "smmnnnooo%", "F000000002", true }, \
+LOG_STRUCTURE_FROM_AIS \
```

Comment:
I think the indenting is off for "LOG_STRUCTURE_FROM_AIS "

Reply:
this matches how the other `#define` ones are done further up the file.